### PR TITLE
Would be proud to join the alliance with my colleagues to represent EDDITS

### PR DESCRIPTION
**Pull request by fcroiseaux**
Correct missing i in Origin url. Added EDDITS project and developers.
Our goal is to link verified identity to ERC725 Identity in a trustless way. We have already linked the Luxembourger Digital Id provider Luxtrust with our solution and have already developed a link with Estonian eID. We are currently working on a solution that will allow any holder of an electronic passport to link it to an ERC725 compliant identity.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -231,7 +264,7 @@ <h2>Projects</h2>
 				</a>
 			</div>
 			<div class="col-12 col-sm-4 col-md-3 col-lg-2 logo text-center">
-				<a href="https://www.orginprotocol.com" target="_blank">
+				<a href="https://www.originprotocol.com" target="_blank">
```
Comment (joshfraser) [inline index.html:340]: Good catch!

**Review by joshfraser — COMMENTED** [review]
Do you happen to have Github URLs you can use instead? Trying to keep the focus on developer efforts & not just marketing.

**Comment by fcroiseaux** [comment]
True for the github repo. I've just migrated the code in https://github.com/Eddits-org

**Comment by fcroiseaux** [comment]
Did you have a look at the change we have made ? I’ve changed the website
url linked to the project icon to the github url of the project

Thanks

-- 
Fabrice Croiseaux

Le 31 août 2018 à 21:28:21, Josh Fraser (notifications@github.com) a écrit:

*@joshfraser* commented on this pull request.

Do you happen to have Github URLs you can use instead? Trying to keep the
focus on developer efforts & not just marketing.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/ERC725Alliance/erc725alliance.github.io/pull/10#pullrequestreview-151523395>,
or mute the thread

**Comment by joshfraser** [comment]
I actually meant Github profiles for the developers. The eddits.io URL was fine.

**Comment by fcroiseaux** [comment]
Done.

**Pull request merged**